projects/groceries/submissions.csv

```diff
@@ -1,3 +1,4 @@
 github_username, repository_url
 my_user, https://github.com/my_user/my_repo
 etc., etc.
+yingluwang, https://github.com/yingluwang/nyu-info-2335-70-201706/edit/master/projects/groceries/submissions.csv
```

Comment:
@yingluwang  Sorry if the instructions weren't clear, but please create a separate GitHub repository (not your fork of the course repository) to contain your work, and then submit the URL to that repository here next time you submit.